pkg/apis/monitoring/v1/thanos_types.go

```diff
@@ -243,6 +243,12 @@ type ThanosRulerSpec struct {
 	// operator itself) or when providing an invalid argument the reconciliation will
 	// fail and an error will be logged.
 	AdditionalArgs []Argument `json:"additionalArgs,omitempty"`
+	// RemoteWriteConfig configures Thanos Ruler to send samples to external systems using remote write.
+	// If the list is not empty, Thanos Ruler doesn't expose the Store API for querying data and `the storage is only used for keeping the write-ahead-log (WAL) data.
+	// Note that Thanos Ruler ignores metadata configuration.
+	// See https://thanos.io/tip/components/rule.md/#stateless-ruler-via-remote-write for more details.
+	// +optional
+	RemoteWrite []RemoteWriteSpec `json:"remoteWrite,omitempty"`
```

Comment:
```suggestion
	// RemoteWrite configures Thanos Ruler to send samples to external systems using remote write.
	// If the list is not empty, Thanos Ruler doesn't expose the Store API for querying data and `the storage is only used for keeping the write-ahead-log (WAL) data.
	// Note that Thanos Ruler ignores metadata configuration.
	// See https://thanos.io/tip/components/rule.md/#stateless-ruler-via-remote-write for more details.
	// +optional
	RemoteWrite []RemoteWriteSpec `json:"remoteWrite,omitempty"`
```

Reply:
I'm also not sure if we want to add so much information about ThanosRuler implementation details here. Maybe we could be succinct and just add the link
```suggestion
	// RemoteWrite configures Thanos Ruler to send samples to external systems using remote write.
	// See https://thanos.io/tip/components/rule.md/#stateless-ruler-via-remote-write for more details.
	// +optional
	RemoteWrite []RemoteWriteSpec `json:"remoteWrite,omitempty"`
```